packages/app/src/specs/SpecsList.vue

```diff
@@ -129,15 +246,48 @@ import SpecItem from './SpecItem.vue'
 import { useVirtualList } from '@packages/frontend-shared/src/composables/useVirtualList'
 import NoResults from '@cy/components/NoResults.vue'
 import SpecPatternModal from '../components/SpecPatternModal.vue'
-import { useDebounce } from '@vueuse/core'
+import { useDebounce, useOnline, useResizeObserver } from '@vueuse/core'
 import Alert from '../../../frontend-shared/src/components/Alert.vue'
 import InlineCodeFragment from '../../../frontend-shared/src/components/InlineCodeFragment.vue'
 import WarningIcon from '~icons/cy/warning_x16.svg'
+import RefreshIcon from '~icons/cy/action-restart_x16'
 import { useRoute } from 'vue-router'
+import { CloudData_RefetchDocument } from '../generated/graphql-test'
+import type { RemoteFetchableStatus } from '@packages/frontend-shared/src/generated/graphql'
 
 const route = useRoute()
 const { t } = useI18n()
 
+const isOnline = useOnline()
+const isOffline = ref(false)
+
+watch(isOnline, (newIsOnlineValue) => isOffline.value = !newIsOnlineValue)
```

Comment:
Is this really necessary? Seems like we're just using it to control display of the offline banner, so couldn't we just do a `!isOnline` check?

Reply:
Let me clarify my question - could we just use `v-if="!isOnline"` to control that modal and dispense with the logic around `isOffline`?

Reply:
it would work for `v-if` but it won't for `v-model` as that should be a changable value used to track dismissal.